sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -109,6 +110,10 @@ export class EducationProgramController {
       eslEligibility: program.eslEligibility,
       hasJointInstitution: program.hasJointInstitution,
       hasJointDesignatedInstitution: program.hasJointDesignatedInstitution,
+      programIntensity:
+        program.programIntensity === ProgramIntensity.fullTimePartTime
+          ? "yes"
```

Comment:
Please avoid doing it on API. It is a UI matter that could be resolved on Form.IO definition assigning the proper values to options on Form.IO.